gammapy/utils/scripts.py

```diff
@@ -185,25 +185,38 @@ def make_path(path):
     return Path(expandvars(str(path)))
 
 
-def recursive_update_dict(old, new):
-    """Recursively update a dict with new values
+def recursive_merge_dicts(a, b):
+    """Recursively merge two dictionaries, entries in b override entries in a.
 
     The built-in update function cannot be used for hierarchical dicts
-    where only a subset of keys shall be updated.
-    see: http://stackoverflow.com/questions/3232943/update-value-of-a-nested-dictionary-of-varying-depth
+    see:http://stackoverflow.com/questions/3232943/update-value-of-a-nested-dictionary-of-varying-depth/3233356#3233356
```

Comment:
Does this create a clickable link in the HTML output?
I think you have to put a space after the `see:`, no?